database/factories/JobPosterFactory.php

```diff
@@ -45,6 +45,7 @@
     ];
     return [
         'job_term_id' => JobTerm::inRandomOrder()->first()->id,
+        'chosen_lang' => $faker->randomElement($array = array('en', 'fr')),
```

Comment:
That's a really verbose way of creating an array. 😆 `['en', 'fr']` should work fine.

Reply:
Haha oops my bad. I just took from the faker docs to see if it worked and didn't clean it up.